### ISSUE TITLE
统计标签和分类数量的时候会把草稿里的文章也统计进去

### ISSUE THREAD

**Issue by youngjuning** [iissnan/hexo-theme-next#1938]
### I agree and want to create new issue 

- [] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [] Yes, I was on [NexT Documentation Site](http://theme-next.iissnan.com/).
- [] And yes, I already searched for current [issues](https://github.com/iissnan/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Steps to reproduce the behavior 
1. N/A
2. N/A
3. N/A

* Link to demo site with this issue: N/A
* Link(s) to source code or any usefull link(s): N/A

### Hexo Information

```

```

### NexT Information

**NexT Version:**

```
[] Latest Master branch.
[] Latest Release version.
[] Old version - 
```

**NexT Scheme:**

```
[] All schemes
[] Muse
[] Mist
[] Pisces
[] Gemini
```

**Comment by youngjuning** [comment]
www.wakeuptocode.me/categories/，Just like this

**Comment by youngjuning** [comment]
www.wakeuptocode.me/categories/

**Comment by leviding** [comment]
@youngjuning 最好把 Issue Temple 的内容补充完整。然后问题用英文描述一下吧。

**Comment by youngjuning** [comment]
@leviding 这不是中国人开发的吗？

**Comment by leviding** [comment]
@youngjuning 现在是一位外国朋友管理

**Comment by HarlanZhou** [comment]
@youngjuning  您好您是如何被百度收录的博客，是用的自动推送的方法么？

**Comment by youngjuning** [comment]
badui-generate那个插件

**Comment by ivan-nginx** [comment]
Don't understand u, write in English if not solved. If solved, close it please. Thank's!

**Comment by zhuzhuyule** [comment]
@ivan-nginx  his means is like [issues #1685](https://github.com/iissnan/hexo-theme-next/issues/1685), this is bug wait to solve.

**Comment by youngjuning** [comment]
Thanks for your translate

**Comment by ivan-nginx** [comment]
@zhuzhuyule thank's for translate.
@youngjuning this issue will be closed die duplicate of #1685.
